NodeOpera/Dockerfile.txt

```diff
@@ -0,0 +1,42 @@
+USER root
+
+#=========
+# Opera
+#=========
+ENV OPERA_VERSION=60.0.3255.170
+ENV OPERA_DRIVER_VERSION=v.2.45
+
+RUN wget -O- http://deb.opera.com/archive.key | apt-key add - && \
+     echo 'deb http://deb.opera.com/opera-stable/ stable non-free' > /etc/apt/sources.list.d/opera-blink.list
+
+RUN apt-get update
+
+RUN apt-get update && \
+     apt-get -y --no-install-recommends install opera-stable && \
+     opera --version && \
+     rm -Rf /tmp/* && \
```

Comment:
why do you need to clean /tmp?

Reply:
Deleted. It's leftover from different approach how to install opera.